index.html

```diff
@@ -662,6 +662,19 @@ <h2>Algorithms</h2>
 and validate status lists as described by this document.
     </p>
 
+    <p class="issue atrisk"
+       title="Attempting alignment with IETF Token Status List specification">
+The Working Group is seeking feedback related to implementer desire to align
+with the IETF OAuth Working Group
+<a href="https://datatracker.ietf.org/doc/draft-ietf-oauth-status-list/">
+Token Status List</a> specification. If there is interest,
+and to the extent possible, this specification might align
+more closely with the Token Status List bitstring format, as it could be
+beneficial to have the same code base able to process bitstring values from
+both lists. If there is implementer support for these changes, these changes
+might be made during the Candidate Recommendation phase.
```

Comment:
As best I can tell, `bit string` is [correct](https://en.wikipedia.org/wiki/Bit_string); `bitstring` is incorrect. (I could find neither in the linked IETF drafts.) Changing to `bit string` throughout should probably become an issue and/or PR of its own.
```suggestion
       title="Attempting alignment with IETF OAuth Status List specification">
The Working Group is seeking feedback related to implementer desire to align
with the IETF OAuth Working Group
<a href="https://datatracker.ietf.org/doc/draft-ietf-oauth-status-list/">
OAuth Status List</a> specification (formerly the
<a href="https://datatracker.ietf.org/doc/draft-looker-oauth-jwt-cwt-status-list/01/">
JWT and CWT Status List</a> specification). If there is interest,
and to the extent possible, this specification might align
more closely with the OAuth Status List bit string format, as it could be
beneficial to have one codebase able to process bit string values from
both lists. If there is implementer support for such changes, they
might be made during the Candidate Recommendation phase.
```

Reply:
I did a bit of research on this before settling on "bitstring". To be sufficiently pedantic, both forms are acceptable:

https://en.wiktionary.org/wiki/bitstring
https://csrc.nist.gov/glossary/term/bitstring
https://csrc.nist.gov/glossary/term/bit_string
https://en.wikipedia.org/wiki/Bitstring

However, "bitstring" is /less commonly/ used:

https://books.google.com/ngrams/graph?content=bitstring%2Cbit+string&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3

... and one of the things I was optimizing for was making it easier to find this specification given a search instead of being lost in a sea of CS literature. 

Thus, decided on "bitstring" and have been attempting to use it consistently throughout the specification. There is also the notion that we would have to overhyphenate "vc-bit-string-status-list" and camel case feels a bit much with the space as well: `BitStringStatusListCredential`.

These are admittedly weak arguments for "bitstring". The arguments for "bit string" are equally weak. I blame the English language. :)